docs/BR.md

```diff
@@ -1008,6 +1065,68 @@ CAs are permitted to treat a record lookup failure as permission to issue if:
+- *Effective September 15, 2024*, the CA MUST implement Multi-Perspective Issuance Corroboration as described in this section; however, the CA MAY proceed with certificate issuance if either the Primary Domain Validation Determination or Primary CAA Determination is not corroborated as defined in the "# of Network Perspectives and Quorum Requirements" table.
+- *Effective March 15, 2025*, the CA MUST implement Multi-Perspective Issuance Corroboration as described in this section. The CA MUST NOT proceed with certificate issuance if either the Primary Domain Validation Determination or Primary CAA Determination is not corroborated as defined in the "# of Network Perspectives and Quorum Requirements" table and the "Defining Corroborating CAA Evidence" table.
+
+Even though the "# of Network Perspectives and Quorum Requirements" table permits issuance when some Network Perspectives do not corroborate the Primary Domain Validation Determination and Primary CAA Determination, the set of locations of the Network Perspectives that do corroborate the Primary Domain Validation Determination and Primary CAA Determination MUST fall within the service regions of at least two (2) distinct regional Internet registries for a CA to proceed with certificate issuance after September 15, 2025.
+
+Table: # of Network Perspectives and Quorum Requirements
+
+| # Network Perspectives used | Description of Configuration and Quorum Requirements | Permitted Use |
+|--- |------------------------------ |------------------------------ |
+| 2  | This configuration relies on two (2) Network Perspectives.<br><br>When this configuration is used, at least one (1) of the Network Perspectives MUST corroborate the Primary Domain Validation Determination and Primary CAA Determination. | This configuration MUST NOT be used after 9/15/2025. |
+| 3  | This configuration relies on three (3) Network Perspectives.<br><br>When this configuration is used, at least two (2) of the Network Perspectives MUST corroborate the Primary Domain Validation Determination and Primary CAA Determination.  | This configuration MUST NOT be used after 3/15/2026. |
+| 4  | This configuration relies on four (4) Network Perspectives.<br><br>When this configuration is used, at least three (3) of the Network Perspectives MUST corroborate the Primary Domain Validation Determination and Primary CAA Determination. | This configuration MUST NOT be used after 9/15/2026. |
+| 5  | This configuration relies on five (5) Network Perspectives.<br><br>When this configuration is used, at least four (4) of the Network Perspectives MUST corroborate the Primary Domain Validation Determination and Primary CAA Determination. | - |
+| 6+ | This configuration relies on six (6) or more (i.e., "N") Network Perspectives.<br><br>When this configuration is used, at least N-2 Network Perspectives MUST corroborate the Primary Domain Validation Determination and Primary CAA Determination. | - |
```

Comment:
We have three kinda-separate, kinda-continuous timelines here:
1) The "how fully implemented" timeline, presented as a bulleted list, from 2024/03/15 through 2025/03/15
2) The "how far apart" timeline, presented as a standalone paragraph, with one date on 2025/09/15
3) The "how many perspectives" timeline, presented as a table, with dates from 2025/09/15 through 2026/09/15

Honestly I think it would be great for this to be all one timeline, with consistent presentation and monotonic dates. Perhaps something like this:

```suggestion
Phased Implementation Timeline:
- *Effective March 15, 2024*, the CA SHOULD implement Multi-Perspective Issuance Corroboration as described in this section. 
- *Effective September 15, 2024*, the CA MUST implement Multi-Perspective Issuance Corroboration as described in this section. However, the CA MAY proceed with certificate issuance if the number of additional Network Perspectives which do not corroborate either the Primary Domain Validation Determination or Primary CAA Determination is greater than the number allowed in the "Quorum Requirements" table. The CA MUST attempt to corroborate the primary determinations from at least two (2) additional Network Perspectives.
- *Effective March 15, 2025*, the CA MUST NOT proceed with certificate issuance if the number of additional Network Perspectives which do not corroborate either the Primary Domain Validation Determination or Primary CAA Determination greater than the number allowed in the "Quorum Requirements" table.
- *Effective September 15, 2025*, the CA MUST attempt to corroborate the primary determinations from at least three (3) additional Network Perspectives. Additionally, the Network Perspectives that do corroborate the primary determinations MUST fall within the service regions of at least two (2) distinct regional Internet registries.
- *Effective March 15, 2026*, the CA MUST attempt to corroborate the primary determinations from at least four (4) additional Network Perspectives.
- *Effective September 15, 2026*, the CA MUST attempt to corroborate the primary determinations from at least five (5) additional Network Perspectives.

Table: Quorum Requirements

| # Network Perspectives used | # of Allowed non-Corroborations |
| --- | --- |
| 2-5 | 1 |
| 6+ | 2 |
```

Maybe this is way too big of a formatting change for this late in the process. But I thought it was worth considering anyway. What do you think?

Reply:
Thanks, @aarongable!

I think this is a helpful simplification. I'll accept this suggestion, and probably tweak things a bit further (for example, to promote consistent use of "primary determinations" vs "Primary Domain Validation Determination or Primary CAA Determination.")

Will leave discussion open for visibility.